include/pidfd-utils.h

```diff
@@ -0,0 +1,23 @@
+#ifndef UTIL_LINUX_PIDFD_UTILS
+#define UTIL_LINUX_PIDFD_UTILS
+
+#if defined (__linux__)
```

Comment:
&& define(SYS_pidfd_send_signal)

.. it's possible that you compile against old kernel headers where is no SYS_podfd_* at all.